modules/Terminal-Basics/exercises/Customizing-Your-Shell.md

```diff
@@ -24,22 +24,7 @@ Changing environment variables in the shell only applies to the current terminal
 export PATH=$PATH:~/your-custom-directory/
 ```
 
-Another useful environment variable you'll want to add to your `.bashrc` is your custom editor. This variable will tell bash which program to use when you want to open Javascript files.
-
-```bash
-export EDITOR='atom'
```

Comment:
`export EDITOR='atom'` and `export GIT_EDITOR='atom'` is essential to using git without having fot know `vim` so I want everyone to have it. 

when you run `git commit` it opens your `$EDITOR` OR `$GIT_EDITOR` so its nice to have that be your default / preferred editor.

Reply:
@flyrightsister do you still want to merge this?